deps/rust/cargo.bzl

```diff
@@ -21,6 +21,8 @@ PACKAGES = {
     "pico-args": crate.spec(version = "0"),
     "proc-macro2": crate.spec(version = "1"),
     "quote": crate.spec(version = "1"),
+    "ruff_python_ast": crate.spec(git = "https://github.com/astral-sh/ruff.git", tag = "0.7.0"),
+    "ruff_python_parser": crate.spec(git = "https://github.com/astral-sh/ruff.git", tag = "0.7.0"),
```

Comment:
So if I say `tag = "0.7"` will it automatically pick `"0.7.1"` over `0.7.0`?

Reply:
As far as I understand, yes. And "0" will pick up "0.8" too. Please note, it will pick up only after you run `update-rust`. The version will remain locked in the `.lock` file.

We decided to keep single-digit dependencies and run on the treadmill since we don't have any `cargo vet` setup.

I suggest to put "0" too and lock it down further only if upgrade breaks us and there are really good reasons to stay behind.

Reply:
Okay done.